catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/utils/CatalogUtils.java

```diff
@@ -36,27 +46,80 @@ public class CatalogUtils {
 
   private CatalogUtils() {}
 
+  @VisibleForTesting
+  public static PaimonBackendCatalogWrapper loadCatalogBackend(PaimonConfig paimonConfig) {
+    Map<String, String> allConfig = paimonConfig.getAllConfig();
+    AuthenticationConfig authenticationConfig = new AuthenticationConfig(allConfig);
+    if (authenticationConfig.isSimpleAuth()) {
+      return new PaimonBackendCatalogWrapper(loadCatalogBackendWithSimpleAuth(paimonConfig), null);
+    } else if (authenticationConfig.isKerberosAuth()) {
+      Configuration configuration = new Configuration();
+      allConfig.forEach(configuration::set);
+      configuration.set(HADOOP_SECURITY_AUTHORIZATION, "true");
+      configuration.set(HADOOP_SECURITY_AUTHENTICATION, "kerberos");
+
+      try {
+        KerberosClient kerberosClient = new KerberosClient(allConfig, configuration);
+        File keytabFile =
+            kerberosClient.saveKeyTabFileFromUri(UUID.randomUUID().toString().replace("-", ""));
+        kerberosClient.login(keytabFile.getAbsolutePath());
```

Comment:
When will we close the `kerberosClient` and release the resource?

Reply:
https://github.com/apache/gravitino/pull/4075/files#diff-6bbc0b99d3fe56c11f9ba3a5a77bc510fc3a9dcb6769167b4b32ab0b3e92bb6eR51-R54

when `PaimonCatalogOperations` invoke the `close` method.